packages/template-ui/src/views/Home.vue

```diff
@@ -17,7 +17,16 @@
       </b-container>
       <FilterContent v-if="hasFilters" />
 
-      <div v-if="isFilterEmpty">
+
+      <div v-if="hasFlatGrid">
```

Comment:
It looks like this is going to misbehave for a channel with filters. If some filters have been toggled, we'll still show the `<CardGrid>` of `contentNodes` instead of `<FilterResult>`. This might flow better if we put `<FilterResult>` in the first condition, as `v-if="isFilterSet` and then other states follow.

Reply:
Yeah I thought about that. This "flat" option is supposed to also enforce no filters according to the spec https://phabricator.endlessm.com/T32104 . But I thought it was better to avoid option dependencies as that's when things start to get complicated. The question is, should we consider all the option combinations if we are not going to use them? In this case I think you are right, the new option is supposed to only affect the tree of nodes and the home grid.